velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -105,6 +105,99 @@ class PrestoSerializerTest : public ::testing::Test {
     return vectorMaker_->rowVector(childVectors);
   }
 
+  void writeInt32(OutputStream* out, int32_t value) {
```

Comment:
Would it make sense to make it a template to avoid copy-pasting code?

Reply:
done, idk if it turned out better

Reply:
The `write` in facebook::velox::OutputStream only supports `char*` args, writing the value without casting doesn't seem to work.

```
class OutputStream {
  virtual void write(const char* s, std::streamsize count) = 0;
```